translations/setup.tex/he.po

```diff
@@ -126,6 +126,51 @@ msgid ""
 "  \\end{tikzpicture}\n"
 "  \\end{scriptsize}"
 msgstr ""
+"\\begin{scope}[xscale=-1]    \\draw (0, 0) node[inner sep=0] {\\makebox[\\textwidth][c]{\\includegraphics[width=1.25\\linewidth]{\\images/setup.png}}};\n"
```

Comment:
`xscale` is the only change here, everything else remained as it was?

Reply:
I need to actually add the translation, but as regards to the dots location - yes, that's all that was needed

Reply:
So I suggest defining `xscale` in `main_he.tex` as `-1`, and also providing it in `metadata.tex` as `1`, so that it is `1` as default. There will be quite a number of those images, where you will have to copy the same exact text. I suggest defining it, so that you don't have to do that. But, if you insist on copy-pasting, I won't argue.

Reply:
I have no idea how to do it
My knowledge in latex is pretty basic.
it would be nicer that way you suggested, of course.

Reply:
@DarthGandalf, care to help with this?